### PR TITLE
feat: split EPD and GenericEstimate to a hierarchy

### PR DESCRIPTION
**Pull request by alexche-cchange**
Starting from the Preview object to be used in lists, to the normal full object, to Extended object to be used for post-with-refs.

### CODE REVIEW

File: src/openepd/model/epd.py [hunk 1]
```diff
@@ -185,6 +198,30 @@ def validate_includes(cls, v: list | None):
...
+        description="The organization responsible for the underlying LCA (and subsequent summarization as EPD).",
+        default=None,
+    )
+    program_operator: Org | None = pyd.Field(description="JSON object for program operator Org")
+    third_party_verifier: Org | None = pyd.Field(
+        description="JSON object for Org that performed a critical review of the EPD data"
+    )
+
+
+EpdExtended = EpdExtendedV0
```
Comment (corvis) [inline src/openepd/model/epd.py:222]: Extended sounds not very good... maybe `WithDeps` or something else?

**Review by corvis — APPROVED** [review]

File: src/openepd/model/epd.py [hunk 2]
```diff
@@ -19,15 +19,19 @@
...
-from openepd.model.org import Org, Plant
+from openepd.model.lcia import WithLciaMixin
+from openepd.model.org import Org, OrgRef, Plant
 from openepd.model.specs import Specs
 from openepd.model.versioning import OpenEpdVersions, Version
 
 
-class EpdV0(WithAttachmentsMixin, WithAltIdsMixin, BaseDeclaration):
-    """Represent an EPD."""
+class EpdPreviewV0(WithAttachmentsMixin, WithAltIdsMixin, BaseDeclaration):
```
Comment (alexlitvinenko) [inline src/openepd/model/epd.py:28]: Add title, like "EPD (Preview)", "EPD (Full)", "EPD (Create Mode)"?

File: src/openepd/model/epd.py [hunk 3]
```diff
@@ -185,6 +198,30 @@ def validate_includes(cls, v: list | None):
...
 
+class EpdExtendedV0(EpdV0):
+    """
+    Expanded version of the EPD.
+
+    Contains related entities - orgs - with full fields, to support object matching in implementations.
+    """
+
+    manufacturer: Org | None = pyd.Field(
+        description="JSON object for declaring Org. Sometimes called the "
```
Comment (alexlitvinenko) [inline src/openepd/model/epd.py:209]: Reuse description.

**Review by alexlitvinenko — APPROVED** [review]

File: src/openepd/model/epd.py (hunk 2 above)
Comment (alexche-cchange) [reply src/openepd/model/epd.py:28]: Do you mean first line of the docstring?

Comment (alexche-cchange) [reply src/openepd/model/epd.py:35]: Figured out, explicit "title" can be used as a class-constructor kwarg.

**Pull request merged**